masonry_core/src/app/render_root.rs

```diff
@@ -337,7 +337,14 @@ impl RenderRoot {
         };
 
         if let Some(test_font_data) = test_font {
-            let families = root.register_fonts(test_font_data);
+            // We don't use `register_fonts` here because that requests a global relayout.
+            // However, because we are not yet fully initialised (we are before the below call
+            // to `run_rewrite_passes`), `request_layout_all`
```

Comment:
Incomplete doc. I assume the full doc would be "`request_layout_all` would panic".